package.json

```diff
@@ -3,11 +3,28 @@
   "version": "0.0.1",
   "description": "Terminal client for Orbit",
   "main": "index.js",
+  "homepage":"https://github.com/orbitdb/orbit-textui",
+  "bugs":"https://github.com/orbitdb/orbit-textui/issues",
+  "repository":{
+    "type": "https://github.com/orbitdb/orbit-textui"
```

Comment:
I don't think this conforms to https://docs.npmjs.com/files/package.json#repository.

Reply:
you are right, it didn't. It should now